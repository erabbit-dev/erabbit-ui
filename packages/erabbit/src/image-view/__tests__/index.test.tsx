import { ImageView } from '..'
import { mount } from '@vue/test-utils'

test('should render default ImageView', () => {
  const wrapper = mount(ImageView)
  expect(wrapper.html()).toMatchSnapshot()
})
