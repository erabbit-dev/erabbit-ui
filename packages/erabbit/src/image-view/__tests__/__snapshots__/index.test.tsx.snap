// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render default ImageView 1`] = `
"<div class=\\"er-image-view\\">
  <div class=\\"er-image-view__wrapper\\">image-view</div>
</div>"
`;
